event/src/Event/Domain/Command/RegisterEventResult.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+use Prooph\Common\Messaging\Command;
+
+use Event\Domain\Value\EventId;
+use Event\Domain\Value\EventType;
+
+
+final class RegisterEventResult extends Command
+{
+
+    /**
+     * @var \Event\Domain\Value\EventId
+     */
+    private $id;
+
+    /**
+     * @var \Event\Domain\Value\EventType
+     */
+    private $type;
+
+    /**
+     * @var string
+     */
+    private $result;
+
+    private function __construct(EventId $id, EventType $type, string $result)
```

Comment:
`string` should probably be `EventResult` (we have a value type for it)

Reply:
I think `EventType` should not be part of this domain event: it is registered when scheduling.

Reply:
EventResult is now used instead of string for event results and EventType is no longer used